src/plugins/cordova-plugin-inappbrowser/inapp-browser.js

```diff
@@ -0,0 +1,391 @@
+    }
+};
+
+IframeBrowser.prototype.hide = function () {
+    if (this._container && this._isVisible) {
+        this._container.style.display = 'none';
+        this._isVisible = false;
+    }
+};
+
+IframeBrowser.prototype.injectScriptCode = function (callback, args) {
+    if (this._container && this._frame) {
+        var code = args[0].replace(/\r?\n|\r/g, ''),
+            hasCallback = args[1];
+
+        try {
+            var result = this._injectCode(code);
+            if (hasCallback) {
+                callback(result);
+            }
+        } catch (error) {
+            console.error('Error occured while trying to inject script', error);
+        }
+    }
+};
+
+IframeBrowser.prototype.injectScriptFile = function (callback, args) {
+    var file = args[0];
+    var code = '(function () {' +
+        'var head = document.getElementsByTagName("head")[0];' +
+        'var script = document.createElement("script");' +
+        'script.type = "text/javascript";' +
+        'script.id = "inappbrowser-inject-script";' +
+        'script.src = "' + file + '";' +
+        'head.appendChild(script);' +
+        '}());';
+
+    args[0] = code;
+
+    this.injectScriptCode(callback, args);
+};
+
+IframeBrowser.prototype.injectStyleCode = function (callback, args) {
+    if (this._container && this._frame) {
+        var css = args[0].replace(/\r?\n|\r/g, ''),
+            hasCallback = args[1];
+
+        var code = '(function () {' +
+            'var head = document.getElementsByTagName("head")[0];' +
+            'var style = document.createElement("style");' +
+            'style.type = "text/css";' +
+            'if (style.styleSheet) {' +
+            'style.styleSheet.cssText = "' + css + '";' +
+            '} else {' +
+            'style.appendChild(document.createTextNode("' + css + '"));}' +
+            'head.appendChild(style);' +
+            '}());';
+
+        try {
+            var result = this._injectCode(code);
+            if (hasCallback) {
+                callback(result);
+            }
+        } catch (error) {
+            console.error('Error occured while trying to inject style', error);
+        }
+    }
+};
+
+IframeBrowser.prototype.injectStyleFile = function (callback, args) {
+    if (this._container && this._frame) {
+        var file = args[0],
+            hasCallback = args[1];
+
+        var code = '(function () {' +
+            'var head = document.getElementsByTagName("head")[0];' +
+            'var link = document.createElement("link");' +
+            'link.type = "text/css";' +
+            'link.rel = "stylesheet";' +
+            'link.href = "' + file + '";' +
+            'head.appendChild(link);' +
+            '}());';
+
+        try {
+            var result = this._injectCode(code);
+            if (hasCallback) {
+                callback(result);
+            }
+        } catch (error) {
+            console.error('Error occured while trying to inject a style file', error);
```

Comment:
This part is still incomplete, and I will add it in my next commits. I checked at the android implementation and it handles the error of the WebView and notifies.